plugin/adsk/scripts/mayaUsd_createStagesFromExistingLayer.mel

```diff
@@ -92,14 +187,10 @@ global proc mayaUsd_createStagesFromExistingLayer() {
         -okCaption "Create"
         -optionsUICreate "stageFromExistingLayer_UISetup"
         -optionsUIInit "stageFromExistingLayer_UIInit"
-        -optionsUICommit2 "stageFromExistingLayer_UICommit"`;
+        -optionsUICommit "stageFromExistingLayer_UICommit"`;
```

Comment:
We didn't need the "2" version of this flag since that's the one you use when you need the selected file.  We don't actually use this yet, and some of the code was going to be calling it with bogus empty strings, so I changed back to the flag that matches the behaviour for now since it's a bit more clear.